lib/elixir/test/elixir/code_test.exs

```diff
@@ -55,13 +55,37 @@ defmodule CodeTest do
                end)
     end
 
-    test "includes column information when possible" do
+    test "includes column information on unused variables" do
       assert {_, [%{position: {1, 12}}]} =
                Code.with_diagnostics(fn ->
                  quoted = Code.string_to_quoted!("if true do var = :foo end", columns: true)
                  Code.eval_quoted(quoted, [])
                end)
     end
+
+    test "includes column information on unused aliases" do
+      sample = """
+      alias URI
+      """
+
+      assert {_, [%{position: {1, 1}}]} =
+               Code.with_diagnostics(fn ->
+                 quoted = Code.string_to_quoted!(sample, columns: true)
+                 Code.eval_quoted(quoted, [])
+               end)
+    end
+
+    test "includes column information on unused imports" do
+      sample = """
+      import URI
+      """
+
+      assert {_, [%{position: {1, 1}}]} =
+               Code.with_diagnostics(fn ->
+                 quoted = Code.string_to_quoted!(sample, columns: true)
+                 Code.eval_quoted(quoted, [])
+               end)
+    end
```

Comment:
And now I am wondering if we should move these assertions to kernel/warnings_test.exs. But I think for now it will suffice here.